src/components/NewCommentForm.tsx

```diff
@@ -14,24 +110,33 @@ export const NewCommentForm: React.FC = () => {
             name="name"
             id="comment-author-name"
             placeholder="Name Surname"
-            className="input is-danger"
+            className={cn('input', { 'is-danger': formError.nameError })}
+            value={newComment.name}
+            onChange={(e) => setNewComment({
+              ...newComment,
+              name: e.target.value,
+            })}
```

Comment:
make one handler for all form fields and use for each onChange handler in form.
attribute name in input should be equal proper field in new comment objgect.

```
const handleChange = (e: React.ChangeEvent<HtnlInputElement>) => {
   const { name, value } = e.target;
   setNewComment({
     ...newComment,
     [name]: value,
   })
}
```
```suggestion
            onChange={handleChange}
```